### ISSUE TITLE
Delegate  got POMed without any action

### ISSUE THREAD

**Issue by dakk** [LiskHQ/lisk-sdk#6982]
Today my delegate dakk got POMed without any action, I can't figure what happened. I report the full timeline of events:

1. The server running lisk-core lost network connectivity for ~40 minutes; lisk-core remained enabled with forging enabled
2. In the meanwhile the delegate missed two blocks
3. Server come back online; lisk-core was still running with forging enabled, no reboot / restart happened
4. The node tried to forge a new block; a node reported misbehavior https://liskscan.com/transaction/fc8444770e7ae9f532ac00ba9a11e93238538aba8f1acab34306505b4dc2e984 

Unfortunately, I don't have any logs since lisk-core was running without logging enabled. The only facts I'm 100% sure are:
- the node lost network connectivity for about 40 minutes
- the node didn't restart
- no other server enabled forging on the same delegate
- no other script re-enabled forging
- disk wasn't full

**Comment by ishantiw** [comment]
@dakk 
We tried to reproduce the problem by creating the same conditions as described in the issue, I tested it 3 times with misbehaviour plugin enabled and with several forging nodes simulating mainnet. Our findings,

**Offline,**
```console
{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2288,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2287}

{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2293,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2292}

{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2294,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2293}
```
**Online again,**
```console
{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2781,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2294

{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2841,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2781

{"address":"0ada6a2f6c8f891769366fc9aa6fd9f1facb36cf","forging":true,"height":2939,"maxHeightPrevoted":1395,"maxHeightPreviouslyForged":2841}
```
It seems like it doesn’t affect forger’s info when the node comes back online and doesn’t get PoMed. 
Since the logs are missing from your end, its really hard for us to know what really happened on your node when it was offline. We haven't received any other issues similar to yours and we also tried to reproduce it internally in a network setting as described by you so I assume something malfunctioned on your node for some reason unrelated to Lisk core.
In case if we receive any similar problem again we will reopen this issue.

**Comment by dakk** [comment]
Yep, after further investigation I detected a disk failure with consequent data corruption. I'm pretty sure that was the true problem that generated the Pom
